packages/qwik/src/core/v2/use-store.unit.tsx

```diff
@@ -144,6 +146,76 @@ Error.stackTraceLimit = 100;
         </Component>
       );
     });
+    it('should update value for issue 5597 - setInterval', async () => {
```

Comment:
can you create a seperatee `describe('regression'` section and add it as `it('#5997 - issue description`